views/js/runner/plugins/tools/highlighter/highlighter.js

```diff
@@ -115,6 +115,19 @@ define([
             }
         });
 
+        // iOS devices clears selection after click on button,
+        // so we store prev selection for this devices to be able
+        // to use it after click on highlight button
+        if (/iPad|iPhone|iPod/.test(navigator.userAgent) && !window.MSStream) {
+            var prevSelection;
```

Comment:
Could you please move `var prevSelection` to the top to be aligned to our ESLint coding standards?